modules/preview-react/segmented-control/lib/SegmentedControlList.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+import {
+  createSubcomponent,
+  ExtractProps,
+  createElemPropsHook,
+  composeHooks,
+} from '@workday/canvas-kit-react/common';
+import {Stack} from '@workday/canvas-kit-react/layout';
+import {useListRenderItems, useOverflowListMeasure} from '@workday/canvas-kit-react/collection';
+import {useSegmentedControlModel} from './hooks/useSegmentedControlModel';
+
+export interface SegmentedControlListProps<T = any>
+  extends Omit<Partial<ExtractProps<typeof Stack, never>>, 'children'> {
+  'aria-label': string;
+  children: ((item: T) => React.ReactNode) | React.ReactNode;
+}
+
+const useSegmentedControlList = composeHooks(
+  useOverflowListMeasure,
+  createElemPropsHook(useSegmentedControlModel)(({state: {orientation, disabled}}) => {
+    return {
+      flexDirection: orientation === 'vertical' ? 'column' : 'row',
+      style: {
+        opacity: disabled ? 0.4 : 1,
```

Comment:
do we need to have different pointers for disabled list?

Reply:
no, we don't need. Buttons have `disabled` state so pointer and click event disabled too.